### ISSUE TITLE
HMNetModel cannot be load

### ISSUE THREAD

**Issue by ismu** [Yale-LILY/SummerTime#75]
Hi :)

I have problem with HMNetModel. I can't load it on yours Colab notebook.
I prepare code for this task:

```
from model import HMNetModel

hmn = HMNetModel()
```

I get error:
```
usage: ipykernel_launcher.py [-h] [--command COMMAND] [--conf_file CONF_FILE]
                             [--PYLEARN_MODEL PYLEARN_MODEL]
                             [--master_port MASTER_PORT] [--cluster CLUSTER]
                             [--dist_init_path DIST_INIT_PATH] [--fp16]
                             [--fp16_opt_level FP16_OPT_LEVEL] [--no_cuda]
                             [--config_overrides CONFIG_OVERRIDES]
ipykernel_launcher.py: error: unrecognized arguments: -f /root/.local/share/jupyter/runtime/kernel-d260cea4-6d66-4567-bbda-0e4a0fefba13.json
An exception has occurred, use %tb to see the full traceback.

SystemExit: 2
```

Do you know what's happen?  Thanks in advance for your help :)

**Comment by ismu** [comment]
I investigate a bit this problem. It doesn't seem to work just for the colab. When I install SummerTime locally, model is loaded.

**Comment by niansong1996** [comment]
Can you try now since #79 is just merged to solve some issues for HMNet?

**Comment by ismu** [comment]
Yes I test your code and it seems all work from running point of view. One problem is that, the HMNetModel isn't downloading automatically, I had to do it manually - related with #84?
The second problem is I think poor quality of the results from the model. I don't know it is problem with model or with preprocessing of input to model. I try investigate this a bit :)

**Comment by niansong1996** [comment]
#84 is more related to the dataset caching, not models per se. It should be downloading automatically for the first time, if it hasn't been cached before. 

I believe the AMI model is fine-tuned on AMI dialogue data and the new one is pre-trained on a "dialogue-ized" CNN/DM data. I found the latter one to be working better in my case, hence the change. But it should be easy enough to provide options to the users, I'll be on that soon.

**Comment by ismu** [comment]
Yes, I think the option to choose a model would be good :)
